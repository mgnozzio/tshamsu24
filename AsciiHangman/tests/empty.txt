empty
n
o
t
r
e
a
l
l
y
e
m
p
t
y
