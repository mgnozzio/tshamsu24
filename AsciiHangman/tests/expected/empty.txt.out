Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ 
Please guess a letter: N

   ____  
  |    | 
  |    O 
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ _ 
Please guess a letter: O

   ____  
  |    | 
  |    O 
  |    | 
  |      
__|__    
Puzzle to solve: _ _ _ _ _ 
Please guess a letter: T
Puzzle to solve: _ _ _ T _ 
Please guess a letter: R

   ____  
  |    | 
  |    O 
  |   \| 
  |      
__|__    
Puzzle to solve: _ _ _ T _ 
Please guess a letter: E
Puzzle to solve: E _ _ T _ 
Please guess a letter: A

   ____  
  |    | 
  |    O 
  |   \|/
  |      
__|__    
Puzzle to solve: E _ _ T _ 
Please guess a letter: L

   ____  
  |    | 
  |    O 
  |   \|/
  |   /  
__|__    
Puzzle to solve: E _ _ T _ 
Please guess a letter: L

   ____  
  |    | 
  |    O 
  |   \|/
  |   / \
__|__    
Game over! Player 1 wins!
